### ISSUE TITLE
Got Error when run update-attack.py

### ISSUE THREAD

**Issue by dickens88** [mitre-attack/attack-website#341]
I got an error when trying to run `python3 update-attack.py`. the code version is v3.4.3.

![image](https://user-images.githubusercontent.com/8390209/146524936-7654a8f7-3ce1-4c6e-9d24-3dd7418f4180.png)

I got the same error when trying to build it with docker...

**Comment by IanDavila** [comment]
Hi @dickens88, thanks for opening an issue.

I tried testing on the current stable branch but was not able to reproduce the error. 
Steps that I took:
- git pull origin master
- git checkout master
- git pull
- python3 -m venv env
- source env/bin/activate (for macOS) | env/Scripts/activate.bat (Windows)
- pip3 install -r requirements.txt
- python3 update-attack.py

Are you on a different branch perhaps?
